Clinical-Genomics/chanjo2
# Do not multiply mean coverage values by 100 on coverage reports

### This PR adds | fixes:
- Fix #231 

### How to test:
- Evaluate mean gene coverage over demo d4 file and demo bed file with 4 genes genes using d4toold (direct call) and make sure coverage is around 20
- Test mean coverage using this branch

### Expected outcome:
- Mean coverage in report should be 100 times lower

### Review:

This [version](https://semver.org/) is a:
